From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
Date: Thu, 1 Sep 2022 19:24:53 +0300
Subject: [PATCH] speex: Remove date from version for reproducible builds

---
 src/speexdec.c | 2 +-
 src/speexenc.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/speexdec.c b/src/speexdec.c
index bf306a3942297ca52b6792221af62c6f1e3253d0..719d1cc415f3a7c569953a6db593a01d301181e1 100644
--- a/src/speexdec.c
+++ b/src/speexdec.c
@@ -291,7 +291,7 @@ void version()
 {
    const char* speex_version;
    speex_lib_ctl(SPEEX_LIB_GET_VERSION_STRING, (void*)&speex_version);
-   printf ("speexdec (Speex decoder) version %s (compiled " __DATE__ ")\n", speex_version);
+   printf ("speexdec (Speex decoder) version\n", speex_version);
    printf ("Copyright (C) 2002-2006 Jean-Marc Valin\n");
 }
 
diff --git a/src/speexenc.c b/src/speexenc.c
index 52733399b752f2bd5ff0f0dac86e3f3245c2a3c4..96d9456d82ccea4cf02f2cc61ca42f021e03ce5e 100644
--- a/src/speexenc.c
+++ b/src/speexenc.c
@@ -203,7 +203,7 @@ void version()
 {
    const char* speex_version;
    speex_lib_ctl(SPEEX_LIB_GET_VERSION_STRING, (void*)&speex_version);
-   printf ("speexenc (Speex encoder) version %s (compiled " __DATE__ ")\n", speex_version);
+   printf ("speexenc (Speex encoder) version %s\n", speex_version);
    printf ("Copyright (C) 2002-2006 Jean-Marc Valin\n");
 }
 
